### PR TITLE
KIN-387 delete product files when deleting a category

### PR DESCRIPTION
**Pull request by Cristianpan**
[KIN-387](https://loopcrack.atlassian.net/browse/KIN-387) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem
It's necessary to delete the product data when the category to which it belongs is deleted. 

### Solution
Implement logic to delete product data when its category is deleted

### Requirements checklist

- [x] When a category is deleted, the files associated with the products of the category must also be deleted. 

### Testing

Create a new product and associate it with an existing category, that doesn't come from seeders, delete the category, and check the following: 
- [x] That the category and product data have been correctly removed from the db.
- [x] That the category and product files have been correctly removed from the server.

## Demo

[KIN-387]: https://loopcrack.atlassian.net/browse/KIN-387?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/Controllers/CtrlCategory.php [hunk 1]
```diff
@@ -182,18 +182,18 @@ public function updateCategory(string $categoryId)
...
-            FileManager::deleteFolderWithContent(FILES_UPLOAD_DIRECTORY . $categoryFiles['icon']);
+            FileManager::deleteMultipleFoldersWithContent($filesToDelete);
 
             $response = [
                 'title'   => 'Eliminación exitosa',
                 'message' => 'La categoría ha sido eliminada correctamente',
                 'type'    => 'success',
             ];
         } catch (Throwable $th) {
+            dd($th);
```
Comment (CarlosJCD) [inline app/Controllers/CtrlCategory.php:196]: please remove this debug sentence

**Review by CarlosJCD — COMMENTED** [review]
Nice work, just a small detail

**Review by CheloChidoone — APPROVED** [review]
Good Job @Cristianpan, just check Carlos comment and remove things that are not needed like that debug line

**Review by Oswaldo-Chan — APPROVED** [review]

**Review by CarlosJCD — APPROVED** [review]
Thanks for the changes!

**Review by CarlosJCD — APPROVED** [review]

**Pull request merged**